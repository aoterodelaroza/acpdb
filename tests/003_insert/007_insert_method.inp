## check: 007_insert_method.out -a1e-10
## delete: 007_insert_method.db
## labels: regression quick

system rm -f 007_insert_method.db
connect 007_insert_method.db

insert litref bibtex ../zz_source/bsip2.bib
insert method b3lyp/6-31+G*
 gaussian_keyword method=b3lyp/6-31+G*;gbs=
 psi4_keyword method=b3lyp;basis=6-31+G*
 litrefs bsip1 boys1970
 description This is a method.
end
