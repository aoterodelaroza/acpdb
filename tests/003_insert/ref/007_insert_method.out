* SYSTEM: rm -f 007_insert_method.db

* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 007_insert_method.db
Creating skeleton database 

* INSERT: insert data into the database (LITREF)

* INSERT: insert data into the database (METHOD)

